src/transformers/models/segformer/feature_extraction_segformer.py

```diff
@@ -211,3 +214,46 @@ def __call__(
         encoded_inputs = BatchFeature(data=data, tensor_type=return_tensors)
 
         return encoded_inputs
+
+    def post_process_semantic_segmentation(self, outputs, target_sizes: Union[TensorType, List[Tuple]] = None):
+        """
+        Converts the output of [`SegformerForSemanticSegmentation`] into semantic segmentation maps. Only supports
+        PyTorch.
+
+        Parameters:
+            outputs ([`SegformerForSemanticSegmentation`]):
+                Raw outputs of the model.
+            target_sizes (`torch.Tensor` of shape `(batch_size, 2)` or `List[Tuple]` of length `batch_size`, *optional*):
+                Torch Tensor (or list) corresponding to the requested final size (h, w) of each prediction. If left to
+                None, predictions will not be resized.
+
+        Returns:
+            `List[torch.Tensor]`: A list of `torch.Tensor` of length `batch_size`, where each item is a semantic
+            segmentation map of of the corresponding target_sizes entry (if `target_sizes` is specified). Each entry of
+            each `torch.Tensor` correspond to a semantic class id.
```

Comment:
It seems that if no `target_sizes` are specified, a single `torch.Tensor` is returned, whereas if target_sizes is specified, a list is returned.

Could we update this docstring, or make the output consistent?

Reply:
Thanks, I made the output consistent!